components/activities/ActivitiesTable.tsx

```
@@ -0,0 +1,370 @@
+      case 'diaper': return '👶';
+      default: return '📝';
+    }
+  };
+
+  const getActivityDetails = (type: ActivityType, details: ActivityDetails | null): string => {
+    if (!details) return '-';
+    
+    switch (type) {
+      case 'feed':
+        const feedDetails = details as FeedDetails;
+        if (feedDetails.type === 'bottle') {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)